api/urls.py

```diff
@@ -64,4 +65,8 @@
     path('add-feedback', feedback_views.add_feedback, name="add-feedback"),
     path('get-all-user-feedback', feedback_views.get_all_user_feedback, name="get-all-user-feedback"),
     path('get-feedback/<int:feedback_id>', feedback_views.get_feedback, name="get-feedback"),
+
+    # Currency Conversion
+    path('get-currency-conversion-rate/<str:source_currency_code>_<str:target_currency_code>',
```

Comment:
Change it to `get-currency-conversion-rate/<str:source_currency_code>/<str:target_currency_code>`